.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
-* @bosilca
+* @staff
```

Comment:
Should be @ICLDisco/staff

Reply:
The github pull request interface shows the following warning:


This CODEOWNERS file contains errors:
    Unknown owner on line 1: make sure @staff exists and has write access to the repository

    * @staff